css/v202209/symbiota/main.css

```diff
@@ -1,31 +1,8 @@
-/**
-* * Symbiota Redesign - main.css
-* * Version: September 2022
-* * Author: Laura Rocha Prado
-* * Author URI: http://laura.rochaprado.com
-* * License: GNU General Public License v2 or later
-* * License URI: http://www.gnu.org/licenses/gpl-2.0.html 
-*/
-
-/** 
-* * CONTENTS
-* * - Fonts imports
-* * - Global variables
-* * - Global styles
-* * - Utility classes
-* * - Components
-*/
-
-/* Fonts imports
+/* Style and Font imports
```

Comment:
A blurb at the top of this file to clarify what distinguishes it from css/symb/main.css would be helpful here.